nss-day-cohort-48/rare-client-river-rats
# Nwm fist client

#### Changes Made

1. Modified file src/components/auth/Profile.js and functionality.
2. this will allow anyone logged in to see all users listed and their profiles
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout nwm-fist-client
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
   > Example: When user does BLANK, then BLANK should happen.
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
